bindings/native/bindgen.sh

```diff
@@ -0,0 +1,4 @@
+#/bin/bash
```

Comment:
```suggestion
#! /usr/bin/env bash
set -euo pipefail
```

This would be more portable as a build script.